opentelemetry/proto/collector/README.md

```diff
@@ -4,6 +4,5 @@ This package describes the OpenTelemetry collector protocol.
 
 ## Packages
 
-1. `common` package contains the common messages shared between different services.
```

Comment:
Why is `common` removed? It is still a package, right?

Reply:
Oops! No idea how that happened. Changes to that file have been reverted.